quizapp/src/test/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=billennium
```

Comment:
sugerowałbym zostawianie standardowych username i password dla baz developerskich h2 in memory.

Reply:
ok w takim razie username będzie: sa